scripts/language-file-validator.js

```diff
@@ -16,20 +16,17 @@ const whitespacePaddedKeys = [
   'shopping_info_rental_period_duration',
   'shopping_info_watch_window_duration',
   'classification_intro',
-  'classification_divider'
-];
-const emptyKeys = [
-  'classification_outro',
-  'plan_label_owned'
+  'classification_divider',
 ];
+const emptyKeys = ['classification_outro', 'plan_label_owned'];
 
-languageFilenames.forEach(language => {
+languageFilenames.forEach((language) => {
```

Comment:
Is this linter mandated? I dislike the extra brackets for a single arg.

Reply:
That's because of the prettier `arrowParens` option... I'll change that one for you.